Makefile.am

```diff
@@ -37,6 +37,7 @@ uninstall-local:
 
 # manually install some files under another name
 install-data-local:
+	sed -i '/Exec=/c\Exec=@bindir@/geany %F' @top_builddir@/geany.desktop
```

Comment:
Can't you just do this:

```diff
- Exec=geany %F
+ Exec=@bindir@/geany %F
```

In [the `geany.desktop.in` file](https://github.com/geany/geany/blob/d9f8cdbad58d09f0c18ca8acccb49209263018f0/geany.desktop.in#L7) or similar?

I'm pretty sure if you added it to `AC_CONFIG_FILES` it would get the substitutions. But maybe I'm missing something of Geany's Autotools.

Reply:
@codebrainz It's not that simple, because all the `*dir` variables usually expand to placeholders like `${prefix}/bin` and such.  So you need to expand them first, and as mentioned in another comment, it's not super trivial.  Plus, in some situations they can be overridden up until make calls, so it's better to apply them there.

Reply:
Ah ok. I was thinking it like the pkg-config file where do this, but I guess there we want them unexpanded because pkg-config takes care of it.